From b4a9419e2146cc522d67f8adea0d1677af171394 Mon Sep 17 00:00:00 2001
From: Zhang Zekun <zhangzekun11@huawei.com>
Date: Mon, 24 Oct 2022 07:40:03 +0000
Subject: [PATCH 040/595] [Huawei] Renaming __insert_sp_area to insert_sp_area

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 187808

----------------------------------------------

Fix the following CodeCheck2.0 static check warning:
Do not declare or define reserved identifiers. The reserved identifier:
__insert_sp_area

Fix this by renaming __insert_sp_area to insert_sp_area.

Signed-off-by: Zhang Zekun <zhangzekun11@huawei.com>
---
 mm/share_pool.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mm/share_pool.c b/mm/share_pool.c
index 228f0adbce25..6558cc3a99a1 100644
--- a/mm/share_pool.c
+++ b/mm/share_pool.c
@@ -1733,7 +1733,7 @@ int mg_sp_id_of_current(void)
 EXPORT_SYMBOL_GPL(mg_sp_id_of_current);
 
 /* the caller must hold sp_area_lock */
-static void __insert_sp_area(struct sp_mapping *spm, struct sp_area *spa)
+static void insert_sp_area(struct sp_mapping *spm, struct sp_area *spa)
 {
 	struct rb_node **p = &spm->area_root.rb_node;
 	struct rb_node *parent = NULL;
@@ -1904,7 +1904,7 @@ static struct sp_area *sp_alloc_area(unsigned long size, unsigned long flags,
 	spa->device_id = device_id;
 
 	spa_inc_usage(spa);
-	__insert_sp_area(mapping, spa);
+	insert_sp_area(mapping, spa);
 	mapping->free_area_cache = &spa->rb_node;
 	list_add_tail(&spa->link, &spg->spa_list);
 
-- 
2.17.1

